halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/node/DeploymentConfiguration.java

```diff
@@ -45,7 +45,7 @@
    *
    * @see Halconfig#halyardVersion
    */
-  String version = "nightly";
+  String version = "master-latest-unvalidated";
```

Comment:
I mean, how do you know you are in master? This shouldnt be a static string. In fact none of the tag should be. The naming policy should be defined externally.